src/sample/courseReg.fxml

```diff
@@ -113,20 +113,20 @@
                   </Text>
                   <HBox prefHeight="40.0" prefWidth="440.0" AnchorPane.leftAnchor="10.0" AnchorPane.topAnchor="100.0">
                      <children>
-                        <TextField fx:id="searchCourse" prefHeight="25.0" prefWidth="400.0" promptText="Search Course">
+                        <TextField fx:id="searchCourse" onAction="#searchcourse" onKeyTyped="#searchcourse" prefHeight="25.0" prefWidth="400.0" promptText="Search Course">
```

Comment:
What is the `onKeyTyped` and `onAction` refer to the same `#searchcourse`?

Reply:
`onAction` serves as the backup in case `onKeyTyped` is not functioning due to bug. But less likely `onAction` will need to be used.